src/supervisor-api.ts

```diff
@@ -48,7 +33,22 @@ export class SupervisorAPI {
 		this.healthchecks = healthchecks;
 
 		this.api.disable('x-powered-by');
-		this.api.use(expressLogger);
+		this.api.use(
+			morgan(
```

Comment:
Moving it in here allows for automatic type inference to work properly (ie that it's an `express.Request` rather than a plain `http.IncomingMessage`)